.github/workflows/CI.yaml

```
@@ -274,6 +274,13 @@ jobs:
 
       - name: "Test: Sys-calls Wasmi integrity"
         run: ./scripts/gear.sh test syscalls --release
+        
+      - name: "Test: Try runtime migrations"
+        run: |
+          cargo build -p gear-cli --release --features try-runtime
+          ./target/release/gear try-runtime --dev --runtime ./target/release/wbuild/gear-runtime/gear_runtime.wasm on-runtime-upgrade --checks live --uri wss://rpc-node.gear-tech.io:443
```

```suggestion
          ./target/release/gear try-runtime --dev --checks live --uri wss://rpc-node.gear-tech.io:443
```
This will result in using currently built runtime, isn't it?

---

`--runtime existing`:
```
Error: Input("given runtime is NOT compiled with try-runtime feature!")
```